### PR TITLE
Add Metrics-Server Image

### PR DESCRIPTION
**Pull request by UdBe**
Add Kubernetes-Sig/Metrics-Server [Ironbank] Image

### CODE REVIEW

File: community_images/metrics-server/ironbank/image.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+name: metrics-server
+official_name: Metrics-Server IronBank
```
Comment (kanavphull) [inline community_images/metrics-server/ironbank/image.yml:2]: IronBank -> Iron Bank

File: community_images/metrics-server/ironbank/image.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+source_image_provider: Platform One
+source_image_repo: registry1.dso.mil/ironbank/opensource/kubernetes-sigs/metrics-server
+source_image_repo_link: https://registry1.dso.mil/harbor/projects/3/repositories/opensource%2Fkubernetes-sigs%2Fmetrics-server
+source_image_readme: https://repo1.dso.mil/dsop/opensource/kubernetes-sigs/metrics-server/-/blob/development/README.md
+rf_docker_link: rapidfort/metrics-server-ib
+image_workflow_name: metrics-server_ironbank
+github_location: metrics-server/ironbank
+report_url: https://us01.rapidfort.com/app/community/imageinfo/registry1.dso.mil%2Fironbank%2Fopensource%2Fkubernetes-sigs%2Fmetrics-server
+usage_instructions: |
+  $ docker run rapidfort/metrics-server-ib
```
Comment (kanavphull) [inline community_images/metrics-server/ironbank/image.yml:13]: Can we change usage instructions to helm command?

File: community_images/metrics-server/ironbank/image.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    helm:
+      repo: metrics-server
+      repo_url: https://kubernetes-sigs.github.io/metrics-server/
+      chart: metrics-server
+    image_keys:
+      metrics-server-ib:
+        repository: "image.repository"
+        tag: "image.tag"
+    override_file: "overrides.yml"
+    readiness_wait_pod_name_suffix: []
```
Comment (kanavphull) [inline community_images/metrics-server/ironbank/image.yml:40]: Can we please verify if image run succeeds without `readiness_wait_pod_name_suffix` ?
If so, please remove it from here.

File: community_images/metrics-server/ironbank/k8s_coverage.sh [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+# Getting Runtime Parameters
+
+JSON_PARAMS="$1"
+JSON=$(cat "$JSON_PARAMS")
+
+echo "Json Params for k8s coverage = $JSON"
+
+NAMESPACE=$(jq -r '.namespace_name' < "$JSON_PARAMS")
+RELEASE_NAME=$(jq -r '.release_name' < "$JSON_PARAMS")
+POD_NAME=$(kubectl get pod -n "${NAMESPACE}" | grep "${RELEASE_NAME}-[a-z0-9-]*" --color=auto -o)
```
Comment (kanavphull) [inline community_images/metrics-server/ironbank/k8s_coverage.sh:15]: Can you please verify if `--color=auto` is required?
If not, can you please remove it?

File: community_images/metrics-server/ironbank/k8s_coverage.sh [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+kubectl autoscale deployment php-apache --cpu-percent=50 --min=1 --max=10
+
+# Check the HPA status
+
+kubectl get hpa
+
+# Resource Cleanup
+
+kubectl delete service php-apache
```
Comment (kanavphull) [inline community_images/metrics-server/ironbank/k8s_coverage.sh:39]: Can we move cleanup to a separate file and use `trap` command to cleanup?
You can take reference from thanos-ib

**Review by kanavphull — CHANGES_REQUESTED** [review]

File: community_images/metrics-server/ironbank/k8s_coverage.sh (hunk 4 above)
Comment (UdBe) [reply community_images/metrics-server/ironbank/k8s_coverage.sh:15]: removed` --color=auto`

File: community_images/metrics-server/ironbank/image.yml (hunk 1 above)
Comment (UdBe) [reply community_images/metrics-server/ironbank/image.yml:2]: Done

File: community_images/metrics-server/ironbank/image.yml (hunk 3 above)
Comment (UdBe) [reply community_images/metrics-server/ironbank/image.yml:40]: Removed `readiness_wait_pod_name_suffix`

File: community_images/metrics-server/ironbank/image.yml (hunk 2 above)
Comment (UdBe) [reply community_images/metrics-server/ironbank/image.yml:13]: Updated Usage Instructions

**Review by kanavphull — APPROVED** [review]

**Review by AnmolVirdi — APPROVED** [review]

**Pull request merged**